libs/utils/utils.js

```diff
@@ -428,30 +429,16 @@ export function decorateImageLinks(a) {
   const images = a.querySelectorAll('img');
   if (!images.length) return a;
   [...images].forEach((img) => {
-    // Check if the image has an alt tag with a url & pipe - 'https... |'
-    if (!(hasUrlWithPipe(img.alt))) return a;
-    try {
-      // //Mine for URL and alt text
-      const splitText = img.alt.split('|');
-      const textUrl = new URL(splitText.shift().trim());
-      const altText = splitText.join('|').trim();
-      if (altText) img.alt = altText;
-
-      const pic = img.closest('picture');
-      const picParent = pic.parentElement;
-      const aTag = createTag('a', { href: textUrl, class: 'desc-link' }, pic);
-      if (aTag.href.includes('#_blank')) {
-        aTag.setAttribute('target', '_blank');
-        aTag.href = aTag.href.replace('#_blank', '');
-      }
-      picParent.append(aTag);
-      return a;
-    } catch (e) {
-      /* c8 ignore next 3 */
-      console.log('Failed to decorate image links', e.message);
-      return a;
-    }
+    if (!img.alt || !(hasUrlWithPipe(img.alt))) return;
+    const [source, alt] = img.alt.split('|');
+    const textUrl = new URL(source);
+    if (alt) img.alt = alt;
```

Comment:
You might still want to apply a `.trim()`. A value like `a | b` will make the source `a ` and the alt ` b`. Just some extra spaces, but the output might be nicer without them. Also, if the value is `a | `, then an alt is still added as empty. So the alt value might do with both a trim and a length check.